tsstats/templates/index.jinja2

```diff
@@ -49,5 +59,23 @@
   {% endfor %}
   <small>Generated by <a href="https://github.com/Thor77/TeamspeakStats" rel="noopener">TeamspeakStats</a> at {{ creation_time|frmttime }}</small>
 </div>
+
+<script type="text/javascript">
+  document.addEventListener('click', function(event) {
```

Comment:
Hm, okey. Thanks for linking this article, but you could still just add it to the `nav`-object instead of the complete document?